### DISCUSSION TITLE
Exclude creator from Subjects wordcloud

### DISCUSSION THREAD

**Discussion by tprus-luc** [CollectionBuilder/collectionbuilder.github.io#136] [Q&A] [answered]
Currently, CB includes all item creators as options in the wordcloud on the Subjects page. Is it possible to exclude them? The creators of my project don't want their names to show up there.

**Comment by evanwill** [comment, marked as answer]
Option is in your theme.yml, https://collectionbuilder.github.io/cb-docs/docs/theme/subjects/
